rapid7_insightvm/unit_test/test_async_requests.py

```diff
@@ -0,0 +1,29 @@
+from unittest import TestCase
+from komand_rapid7_insightvm.util import async_requests
+import asyncio
+import json
+
+class MockResponse():
+    def __init__(self):
+        self.status = 200
+
+    def text(self):
+        return '{"test_key": "test_value"}'
+
+    async def json(self):
+        return json.loads(self.text())
+
+class MockSession():
+    async def request(self, url, method, **kwargs):
+        return MockResponse()
+
+class TestAsyncRequests(TestCase):
+    def test_async_request(self):
+        loop = asyncio.get_event_loop()
```

Comment:
Any specific reason you're running it by getting the event loop? According to the documentation you should avoid the lower level functions unless absolutely necessary: https://docs.python.org/3.8/library/asyncio-eventloop.html
```
Application developers should typically use the high-level asyncio functions, such as asyncio.run(), and should rarely need to reference the loop object or call its methods. This section is intended mostly for authors of lower-level code, libraries, and frameworks, who need finer control over the event loop behavior.
```

Reply:
This is how me and joey figured out to unit test the asnyc stuff